shelley/chain-and-ledger/formal-spec/IOHKCoverPage.sty

```diff
@@ -18,7 +18,7 @@
   {\@techreporttrue}
   \ProcessOptions
 %
-\def\IOHK{IOHK}
+\def\IOHK{\textbf{IOHK}}
```

Comment:
why bold? I would prefer `\textsc{Iohk}` but that is subjective of course :-)

Reply:
I'm not bothered :)  I'll see what Philipp wants